wallet-test/src/test/scala/org/bitcoins/wallet/dlc/DLCNegotiationTest.scala

```diff
@@ -0,0 +1,101 @@
+package org.bitcoins.wallet.dlc
+
+import org.bitcoins.dlc.DLCMessage
+import org.bitcoins.testkit.wallet.FundWalletUtil.FundedWallet
+import org.bitcoins.testkit.wallet.{BitcoinSWalletTest, DLCWalletUtil}
+import org.scalatest.FutureOutcome
+
+class DLCNegotiationTest extends BitcoinSWalletTest {
+  type FixtureParam = FundedWallet
+
+  override def withFixture(test: OneArgAsyncTest): FutureOutcome = {
+    withFundedSegwitWallet(test)
+  }
+
+  behavior of "DLCNegotiation"
```

Comment:
Should this not be `DLCWallet`? I'm not able to parse `"DLCNegotiation must create a dlc offer"`